js/varien/js.js

```diff
@@ -609,18 +609,6 @@ Element.addMethods({
     }
 });
 
-/*
-if (!("console" in window) || !("firebug" in console))
-{
-    var names = ["log", "debug", "info", "warn", "error", "assert", "dir", "dirxml",
-    "group", "groupEnd", "time", "timeEnd", "count", "trace", "profile", "profileEnd"];
-
-    window.console = {};
-    for (var i = 0; i < names.length; ++i)
-        window.console[names[i]] = function() {}
-}
-*/
-
```

Comment:
This change should be discussed. It is not related to this PR.

Reply:
it is commented code so it's fine to remove it altough not part of the specific scope of the PR